List/LocationList.py

```diff
@@ -367,18 +367,18 @@ def getTreasureList():
 
 
 
-            KH2Treasure(447, "Old Mansion Potion", LocationTypes=[locationType.TT]),
-            KH2Treasure(448, "Old Mansion Mythril Shard", LocationTypes=[locationType.TT]),
-            KH2Treasure(442, "The Woods Potion", LocationTypes=[locationType.TT]),
-            KH2Treasure(443, "The Woods Mythril Shard", LocationTypes=[locationType.TT]),
-            KH2Treasure(444, "The Woods Hi-Potion", LocationTypes=[locationType.TT]),
-            KH2Treasure(420, "Tram Common Hi-Potion", LocationTypes=[locationType.TT]),
-            KH2Treasure(421, "Tram Common AP Boost", LocationTypes=[locationType.TT]),
-            KH2Treasure(422, "Tram Common Tent", LocationTypes=[locationType.TT]),
-            KH2Treasure(423, "Tram Common Mythril Shard (1)", LocationTypes=[locationType.TT]),
-            KH2Treasure(424, "Tram Common Potion (1)", LocationTypes=[locationType.TT]),
-            KH2Treasure(425, "Tram Common Mythril Shard (2)", LocationTypes=[locationType.TT]),
-            KH2Treasure(484, "Tram Common Potion (2)", LocationTypes=[locationType.TT]),
+            KH2Treasure(447, "Old Mansion Potion", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(448, "Old Mansion Mythril Shard", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(442, "The Woods Potion", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(443, "The Woods Mythril Shard", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(444, "The Woods Hi-Potion", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(420, "Tram Common Hi-Potion", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(421, "Tram Common AP Boost", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(422, "Tram Common Tent", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(423, "Tram Common Mythril Shard (1)", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(424, "Tram Common Potion (1)", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(425, "Tram Common Mythril Shard (2)", LocationTypes=[locationType.TT]).setLocationWeight(-10),
+            KH2Treasure(484, "Tram Common Potion (2)", LocationTypes=[locationType.TT]).setLocationWeight(-10),
             KH2Treasure(526, "Station Plaza Secret Ansem Report 2", LocationTypes=[locationType.TT], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
```

Comment:
All of the first visit TT chests should have the same weight, since once you go into TT1 you have to finish it

Reply:
I was worried about the distribution of checks being too biased in TT (I didn't want a 15 check TT1), but maybe that's okay.